src/item.cpp

```diff
@@ -4348,10 +4348,24 @@ bool item::is_power_armor() const
 
 int item::get_encumber( const Character &p ) const
 {
+    units::volume char_storage( 0_ml );
     units::volume contents_volume( 0_ml );
+
     for( const item &e : contents ) {
         contents_volume += e.volume();
     }
+
+    for( const item &e : p.worn ) {
+        char_storage += e.get_storage();
+    }
+    if( char_storage != 0_ml ) {
+        const islot_armor *t = find_armor_data();
+        if( t != nullptr ) {
+            contents_volume += units::from_milliliter( t->storage.value() * p.inv.volume().value() /
```

Comment:
I suspect you need to cast to `int64_t` here, otherwise there's a risk of integer overflow for large backpacks (of ~46l or more, if my back-of-the-envelope calculations are correct).